src/frontend/components/TourGuide/TourGuide.scss

```diff
@@ -0,0 +1,77 @@
+.introjs-tooltip {
+  background: #222530;
+  color: var(--text-default);
+  border-radius: var(--space-xs);
+  box-shadow: 0px 4px 20px rgba(0, 0, 0, 0.25);
+  border: 1px solid var(--accent);
+  min-width: 320px;
+}
+
+.introjs-tooltiptext {
+  font-size: var(--text-xl);
+  line-height: 1.5;
+  padding: var(--space-sm);
+}
+
+.introjs-button {
+  border: 2px solid var(--stroke-brand-strong, #ab82fe);
+  border-radius: var(--space-sm, 8px);
+  background: var(--Fill-Brand-strong, #6b2adc);
+  color: var(--text-strong, #fff);
+  text-align: center;
+  text-shadow: none;
+
+  /* Text/Button-Sm */
+  font-family: var(--font-family-body, Poppins);
```

Comment:
You should use the button variable "small" and remove this style, as the button has the proper style already.

Reply:
I needed to implement the button since introJS doesnt have a way of passing our buttons from hp-ui.